apps/11gist-blog/script.js

```diff
@@ -1,5 +1,30 @@
 'use strict';
 
 $(document).ready(function() {
-  // You code here
+  $.ajax('http://127.0.0.1:8080/apps/11gist-blog/api/gists.json', {
```

Comment:
You can shorten this ajax call to `$.ajax("api/gists.json",` since the api folder is relatively close to the js file. Very similar to how you handle linking css files to an html page.